src/test/java/com/conveyal/gtfs/validator/SpeedTripValidatorTest.java

```diff
@@ -12,8 +12,7 @@
 import static com.conveyal.gtfs.GTFS.load;
 import static com.conveyal.gtfs.GTFS.validate;
 import static com.conveyal.gtfs.TestUtils.assertThatSqlCountQueryYieldsExpectedCount;
-import static com.conveyal.gtfs.error.NewGTFSErrorType.TRAVEL_TOO_FAST;
-import static com.conveyal.gtfs.error.NewGTFSErrorType.TRAVEL_TOO_SLOW;
+import static com.conveyal.gtfs.error.NewGTFSErrorType.*;
```

Comment:
Revert this change and import enum values explicitly (we prefer to avoid mass imports).